pkg/channel/event_dispatcher.go

```diff
@@ -32,7 +32,18 @@ import (
 	"knative.dev/eventing/pkg/utils"
 )
 
-const correlationIDHeaderName = "Knative-Correlation-Id"
+const (
+	correlationIDHeaderName = "Knative-Correlation-Id"
+
+	// TODO make these constants configurable (either as env variables, config map, or part of broker spec).
```

Comment:
should update the comment --> remove broker spec...

Reply:
Done.